tests/st/calicoctl/test_crud.py

```diff
@@ -76,6 +78,19 @@ def test_get(self):
         rc = calicoctl("delete ippool %s" % name(ippool_name2_rev1_v6))
         rc.assert_error(text=NOT_FOUND)
 
+    def test_broke_store_config(self):
+        """
+        Test that broken store configuration does not crash
+        """
+
+        rc = calicoctl("get policy", broken_store=True)
+        rc.assert_error()
+        rc.assert_output_not_contains("panic")
```

Comment:
Do we need to have these `rc.assert_output_not_contains("panic")` calls separately from `rc.assert_error()`?  I think it would make sense for `rc.assert_error()` *always* to check that the output does not include a panic.  WDYT?

Reply:
indeed, done